### DISCUSSION TITLE
缴费邮箱无法收到登录专业版验证码

### DISCUSSION THREAD

**Discussion by weichengcong** [getlantern/lantern-forum-cn#646] [General]
通过weichengcong@163.com缴费2年专业版，至今未收到登录码
![Uploading 微信图片_20240622201911.jpg…]()
![Uploading 微信图片_20240622201911.jpg…]()

**Comment by MaxMKMax** [comment]
@weichengcong 您好. 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
